frontend/src/components/common/sidebar/sidebar.module.scss

```diff
@@ -0,0 +1,144 @@
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  width: 300px;
+  height: 100%;
+  overflow: auto;
+  background-color: var(--gray-300-header-menu-background);
+
+  @media screen and (max-width: 1300px) {
+    width: 64px;
+  }
+
+  @media screen and (max-width: 560px) {
+    display: none;
+  }
+}
+
+.sidebar::-webkit-scrollbar {
+  display: none;
+}
```

Comment:
Why did you decide to force hide scrollbar here?

Reply:
@kyrylo-burmelov because we focus on youtube

Reply:
@artem-drozdov according to today's meeting it has to be changed